### ISSUE TITLE
Create README.md

### ISSUE THREAD

**Issue by deepesh611** [07sumit1002/CabRental#39]
Create a proper README.md, give proper description, instructions on how to use a local copy of this repo, contributors, etc.

**Comment by 07sumit1002** [comment]
@deepesh611  Go ahead and work on it.

**Comment by deepesh611** [comment]
Sure, however I will need a bit of your help.

It would be helpful if u could give a list of points that sum you your project. It may be the best features, the target audience, anything that you can think of, that you would like the public to see.

As a programmer, I could list some things by reading the repo contents, however there is a quite high chance that I may miss some point, and so I turn to the repo owner for help.

**Comment by 07sumit1002** [comment]
Yeah you can find a pdf file in repo most of data can be found there if you still need something do tell me

**Comment by 07sumit1002** [comment]
Yeah you can find a pdf file in repo most of data can be found there if you still need something do tell me.

**Comment by deepesh611** [comment]
I have submitted a PR, pls have a look and make sure to verify the information adn give a feedback
